### PR TITLE
header & footer started

### PR DESCRIPTION
**Pull request by anyakrainiuk**
[DEMO LINK](https://anyakrainiuk.github.io/Museum_2/)

### CODE REVIEW

**Review by danylo-onishchuk — CHANGES_REQUESTED** [review]
давай сдавать готовые блоки со всем адаптивом

эти блоки пока не готовыс

**Comment by anyakrainiuk** [comment]
> 

добре, я просто вирішила щоб видно було,що почала робити.

**Review by dellachaise — CHANGES_REQUESTED** [review]
Nice start 👍 

1. Add favicon
2. Check inserts on desktop, tablet, mobile with mockup.
Screenshot 2022-08-01 at 19 27 18
Screenshot 2022-08-01 at 19 27 55
Screenshot 2022-08-01 at 19 28 26

**Comment by anyakrainiuk** [comment]
Favicon was added

**Comment by dellachaise** [comment]
Nice progress 👍  Keep going 🔥 

Comments about title still actual)

**Comment by anyakrainiuk** [comment]
> Nice progress 👍 Keep going 🔥
> 
> Comments about title still actual)

I am working on it)

**Review by dellachaise — CHANGES_REQUESTED** [review]
Great progress 👍  Looks 🔥 

Please add fixes according to comments:
1. On big screen picture in header shouldn't change its position.
Screenshot 2022-08-03 at 19 47 24
2. Add styles on hover for menu items and address.
Screenshot 2022-08-03 at 19 52 24
3. Form doesn't work properly. You need to add changes to html - it is a form with a button.
On form submitting you need to use [preventDefault](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault) to stop the update page and [reset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset) form

Screenshot 2022-08-03 at 19 53 36
4. Better add some hover effects for social medias.
Screenshot 2022-08-03 at 19 59 12

**Comment by anyakrainiuk** [comment]
> Great progress 👍 Looks 🔥
> 
> Please add fixes according to comments:
> 
> 1. On big screen picture in header shouldn't change its position.
>    Screenshot 2022-08-03 at 19 47 24
> 2. Add styles on hover for menu items and address.
>    Screenshot 2022-08-03 at 19 52 24
> 3. Form doesn't work properly. You need to add changes to html - it is a form with a button.
>    On form submitting you need to use [preventDefault](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault) to stop the update page and [reset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset) form
> 
> Screenshot 2022-08-03 at 19 53 36 4. Better add some hover effects for social medias. Screenshot 2022-08-03 at 19 59 12

Thanks)
Ok, i will fix it.

File: src/styles/blocks.scss/button.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  font-weight: 600;
+  border: none;
+  transition: 0.3s;
```
Comment (dellachaise) [inline src/styles/blocks.scss/button.scss:8]: Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.

Check all over the project.

File: src/styles/blocks.scss/header.scss [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    }
+
+    &-subtitle {
+      font-family: "Raleway", serif;
+      font-weight: 400;
+      font-size: 16px;
+      line-height: 150%;
+      color: #fff;
+      padding-bottom: 20px;
+    }
```
Comment (dellachaise) [inline src/styles/blocks.scss/header.scss:62]: Why. do you use such a nesting here `&-`?

You could nest modifiers, or elements, but there you have non of them.

Watch more about naming convention in BEM [in this video](https://mate.academy/learn/css-advanced/bem#/video/920)
Watch closely when you use '-' in naming.


Please check this moments all over the project.

File: src/styles/blocks.scss/header.scss [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    margin-bottom: 75px;
+
+    &-icons {
+      display: flex;
+      font-display: row;
+
+      &-choose {
+        opacity: 0;
+        background-color: $mainRed;
+        border: none;
```
Comment (dellachaise) [inline src/styles/blocks.scss/header.scss:80]: The same

File: src/styles/blocks.scss/logo.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.logo {
+  &__img {
+    height: 30px;
+    &:hover {
+      transform: scale(1.1);
+    }
+  }
+
+  &__menu-language {
```
Comment (dellachaise) [inline src/styles/blocks.scss/logo.scss:9]: Add transition to `&__menu-language`.

File: src/styles/blocks.scss/logo.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
+.logo {
+  &__img {
+    height: 30px;
```
Comment (dellachaise) [inline src/styles/blocks.scss/logo.scss:3]: Add transition to `&__img`.

And check all over the project that you have transition with property and duration for the element.

**Review by dellachaise — CHANGES_REQUESTED** [review]
Great progress 👍 

Better add changes according to comments:
1. On big screen picture still in header shouldn't change its position.
Try to add `.header__content ` `position: relative;` and lover `    left`  for `header__photo `

Screenshot 2022-08-03 at 19 47 24

File: src/styles/blocks.scss/button.scss (hunk 1 above)
Comment (anyakrainiuk) [reply src/styles/blocks.scss/button.scss]: Знімок екрана 2022-08-05 о 13 57 28
I added position, but not sure about changing left value. Now it looks like this.

**Review by dellachaise — CHANGES_REQUESTED** [review]
Your site looks awesome 🔥  Great job 👍 

Better add underline instead of scale for this menu items
Screenshot 2022-08-05 at 15 21 57
 
 But we have some misunderstanding with code.
 
 1. Please rewatch video for BEM - where we use element, where modifiers. 
 [video](https://mate.academy/learn/css-advanced/bem#/video/909)
 If you would have some questions - let me know.
 
 2. About transition property we also had misunderstanding. Use transition property all, or don't add it(the same as, all, because all - it's default value) it is a bad practice.  With transition property all or without it - it's hard for the browser to control all properties. And it will work not as good.

**Review by dellachaise — APPROVED** [review]
Great job 👍 

Just fix this issues:
1. On all versions this indent(under submit button) should be bigger, check in mockup and fix.
Screenshot 2022-08-05 at 21 05 16

2. At the lest of the footer shouldn't be this picture(on desktop).

**Pull request closed without merging**